cf-agent/verify_files_utils.c

```diff
@@ -836,7 +837,7 @@ static PromiseResult SourceSearchAndCopy(EvalContext *ctx, const char *from, cha
             }
         }
 
-        if (attr.copy.purge)    /* Purge this file */
+        if (attr.copy.purge)    /* Do not purge this file */
```

Comment:
The idea is that `namecache` contains the list of files not to purge, so the comment was quite misleading, but I can put it on the next line to make clearer.